models.py

```diff
@@ -484,12 +291,38 @@ def update_order(self, order_id, order_status, session):
                     filter(CustOrderStatus.order_id == order_id).\
                     update({CustOrderStatus.order_status: order_status}, 
                     synchronize_session=False)
-        try:
-            session.commit()
-            return update
-        except:
-            session.rollback()
-            raise Exception("update not completed!")
-        finally:
-            session.expunge_all()
-            session.close()
\ No newline at end of file
+        session.commit()
+        return update
+
+
+class CommonFunctions:
```

Comment:
You don't need to create a class for the common functions. Just create normal functions.

Reply:
created normal functions